website/blog/2023-05-04-generating-dynamic-docs.md

```diff
@@ -0,0 +1,368 @@
+---
+title: "Generating dynamic dbt docs for a whole domain at once"
+description: "For columns that are reused across models, Mikael walks through a DRY-method to generate dynamic documentation using the dbt Codegen package and docs blocks."
+slug: generating-dynamic-docs-dbt
+
+authors: [mikael_thorup]
+
+tags: [dbt tutorials]
+hide_table_of_contents: false
+
+date: 2023-05-09
+is_featured: true
+---
+
+At [Lunar](https://www.lunar.app/), most of our dbt models are sourcing from event-driven architecture. As an example, we have the following models for our account domain in our ingestion layer:
```

Comment:
It'd be worth explaining providing a definition of "domain" (even just saying it's a folder full of models related to the same thing).

Reply:
Alternatively, you could probably get broader understanding by just replacing the word "domain" with "folder" across the board.